### DISCUSSION TITLE
Bounty Master issue

### DISCUSSION THREAD

**Discussion by westyofoz** [Aelto/W3_RandomEncounters_Tweaks#137] [Q&A] [answered]
First up, this mod is awesome, thanks for the hard work in creating it. I have been looking for something like this. 

Here comes the 'but.' I installed the mod, no problems, no conflicts etc. The options appear to be working with no problems. The bounties appear and I can go kill the beast/s and loot the trophies etc. I can do the crouch thing to find out about the local ecosystem. Everything appears to be working except the Bounty Master. He just says 'Geralt' and that is it. He is appearing with the exclamation on the map and has the circle etc, he just doesn't want to talk to poor old Geralt and reward him.

I searched for something similar but I haven't found any discussion of this particular issue. I tried reinstalling the mod but same issue. Any suggestions on how to fix this inconvenience would be greatly appreciated.

**Comment by Aelto** [comment, marked as answer]
Hello! It could be your NPC interaction event listener is not firing up properly, could you please confirm everything about `sharedutils_npcInteraction` is merged **and every conflict about this module is solved with atleast the sharedutils code included**. You could have the game running without having the section of the sharedutils_npcInteraction that fires the event when you interact with an NPC, causing what you currently experience.

EDIT: the important section i'm talking about is this one: [`components.ws`](https://github.com/Aelto/tw3-shared-utils/blob/master/mod_sharedutils_npcInteraction/content/scripts/engine/components.ws#L135-L139)

**Reply by westyofoz** [reply]
> Hello! It could be your NPC interaction event listener is not firing up properly, could you please confirm everything about `sharedutils_npcInteraction` is merged **and every conflict about this module is solved with atleast the sharedutils code included**. You could have the game running without having the section of the sharedutils_npcInteraction that fires the event when you interact with an NPC, causing what you currently experience.
> 
> EDIT: the important section i'm talking about is this one: [`components.ws`](https://github.com/Aelto/tw3-shared-utils/blob/master/mod_sharedutils_npcInteraction/content/scripts/engine/components.ws#L135-L139)

Thanks Aelto for the reply. There was a conflict with modSharedImports that I had merged. The section of code is included in the merged file. 

I found the issue, it was in the load order. All works fine now. Thank you very much for your help. Appreciated

**Reply by Aelto** [reply]
Good to hear it is now solved, have fun!
